packages/gateway/src/metrics.js

```diff
@@ -68,63 +68,65 @@ export async function metricsGet(request, env, ctx) {
   }
 
   const metrics = [
-    `# HELP nftstorage_gateway_total_cached_responses Total cached responses returned.`,
-    `# TYPE nftstorage_gateway_total_cached_responses counter`,
-    `nftstorage_gateway_total_cached_responses{env="${env.ENV}"} ${metricsCollected.summaryMetrics.totalCachedResponses}`,
-    `# HELP nftstorage_gateway_total_winner_response_time Total requests performed.`,
-    `# TYPE nftstorage_gateway_total_winner_response_time counter`,
-    `nftstorage_gateway_total_winner_response_time{env="${env.ENV}"} ${metricsCollected.summaryMetrics.totalWinnerResponseTime}`,
-    `# HELP nftstorage_gateway_total_winner_successful_requests Total successful requests.`,
-    `# TYPE nftstorage_gateway_total_winner_successful_requests counter`,
-    `nftstorage_gateway_total_winner_successful_requests{env="${env.ENV}"} ${metricsCollected.summaryMetrics.totalWinnerSuccessfulRequests}`,
-    `# HELP nftstorage_gateway_total_response_time Average response time.`,
-    `# TYPE nftstorage_gateway_total_response_time gauge`,
+    `# HELP nftstorage_gateway_total_cache_hit_responses Total cached responses returned.`,
+    `# TYPE nftstorage_gateway_total_cache_hit_responses counter`,
+    `nftstorage_gateway_total_cache_hit_responses{env="${env.ENV}"} ${metricsCollected.summaryMetrics.totalCachedResponses}`,
```

Comment:
Should these be `nftgateway` and suffixed with `total`?

Reply:
Yes, sorry! I don't know why I did not change these lol